validation/keysin_test.go

```diff
@@ -0,0 +1,72 @@
+package validation
+
+import "testing"
+
+func TestKeysInValidator_Validate(t *testing.T) {
```

Comment:
Please add a test for the validator constructor (checking it's `Name()`, `IsType()`, `IsTypeDependent()` and `MessagePlaceholders()`.